tests/auxil/bot_method_checks.py

```diff
@@ -62,6 +70,33 @@ def check_shortcut_signature(
     Returns:
         :obj:`bool`: Whether or not the signature matches.
     """
+
+    def resolve_class(name: str) -> Optional[type]:
+        """Attempts to resolve a PTB class from a ForwardRef.
+
+        E.g. resolves <class 'telegram._files.sticker.StickerSet'> from "StickerSet".
+
+        Returns a class on success, :obj:`None` if nothing could be resolved.
+        """
+
+        for option in (
+            name,
+            f"telegram.{name}",
```

Comment:
```suggestion
            f"telegram.{name}",
            f"telegram.request.{name}",
```
Do we need to include the other modules? This function only checks bot methods isn't it? So we don't need to check ext.filters either correct?

Reply:
OK :)